<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="TripService Get Calls"
           tests="2"
           failures="1"
           errors="0"
           skipped="1"
           time="0.798">
  <properties />
            <testcase name="getProducttypeBookingType" time="0.280" classname="tripServicesProd.TS_Get_ProductType_BookType">
                <failure type="java.lang.AssertionError"
                  message="expected [true] but found [false]"
              >
<![CDATA[java.lang.AssertionError: expected [true] but found [false]
  org.testng.Assert.fail(Assert.java:96)
  org.testng.Assert.failNotEquals(Assert.java:776)
  org.testng.Assert.assertTrue(Assert.java:44)
  org.testng.Assert.assertTrue(Assert.java:54)
  tripServicesProd.TS_Get_ProductType_BookType.getProducttypeBookingType(TS_Get_ProductType_BookType.java:41)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
  org.testng.internal.Invoker.retryFailed(Invoker.java:839)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  java.lang.Thread.run(Thread.java:748)
]]>
      </failure>

          </testcase>
  
            <testcase name="getProducttypeBookingType" time="0.0" classname="tripServicesProd.TS_Get_ProductType_BookType">
          <skipped />
    </testcase>
  
  
</testsuite>
